apps/dashboard/src/contract-ui/tabs/shared-components/marketplace-table.tsx

```diff
@@ -162,10 +161,7 @@ export const MarketplaceTable: React.FC<MarketplaceTableProps> = ({
       },
       manualPagination: true,
       pageCount: Math.max(
-        Math.ceil(
-          BigNumber.from(totalCountQuery.data || 0).toNumber() /
-            queryParams.count,
-        ),
+        Math.ceil(Number(totalCountQuery.data || 0n) / queryParams.count),
```

Comment:
you could `max(totalCountQuery.data || 0n, Number.MaxSafeInterger)` to be safe that totalCountQuery will never go over, but probably over-optimzation here